src/components/base/OakCloudinaryImage/OakCloudinaryImage.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { CldImage, CldImageProps } from "next-cloudinary";
+
+import { OakImage, OakImageProps } from "../OakImage";
+
+import { getCloudinaryConfig, getCloudinaryIdFromUrl } from "./cloudinary";
+
+export type OakCloudinaryImageProps = Omit<
+  OakImageProps<typeof CldImage>,
+  "src" | "imageProps"
+> & {
+  /**
+   * Override the global cloudinary config set with `setCloudinaryConfig` for this image
+   */
+  config?: CldImageProps["config"];
+  /**
+   * The cloudinary image id or the full cloudinary URL
+   *
+   * Usually in the format `v1234567890/image.jpg`
+   */
+  cloudinaryId: string;
+};
+
+/**
+ * OakCloudinaryImage wraps OakImage providing responsive images from Cloudinary
+ * based on the `sizes` prop.
+ *
+ * Cloudinary config can be set globally with `setCloudinaryConfig` or per image with the `config` prop.
+ */
+export const OakCloudinaryImage = ({
+  config: incomingConfig,
+  cloudinaryId,
+  unoptimized = false || cloudinaryId.endsWith(".svg"),
+  ...props
+}: OakCloudinaryImageProps) => {
+  const config = incomingConfig ?? getCloudinaryConfig();
```

Comment:
This allows us to serve images from multiple Cloudinary instances (in our case the assets or quiz images).

a global/default config can be set with `setCloudinaryConfig` or a prop passed with a specific config for a given image.